java/src/main/native/OrtJniUtil.h

```diff
@@ -11,6 +11,11 @@
 extern "C" {
 #endif
 
+static const char * const ORTJNI_StringClassName = "java/lang/String";
+static const char * const ORTJNI_OnnxValueClassName = "ai/onnxruntime/OnnxValue";
+static const char * const ORTJNI_NodeInfoClassName = "ai/onnxruntime/NodeInfo";
+static const char * const ORTJNI_MetadataClassName = "ai/onnxruntime/OnnxModelMetadata";
```

Comment:
If you wanted them to be shared across the files, you would want a single non-static copy, rather than creating a separate copy for all files that include this header.
Declare them extern const char* without assigning a value.
Place the actual initializations into OrtJniUtil.c file.

Reply:
Ok.